### PR TITLE
Update asc & binaryen versions and fix nearlib API

### PR DESCRIPTION
**Pull request by willemneal**

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -258,8 +258,8 @@ export class App extends React.Component<AppProps, AppState> {
     if (!this.state.accountId) {
       const randomSuffix = Math.floor(Math.random() * 9999999999);
       const accountId = "studio" + randomSuffix;
-      const keyPair = await KeyPair.fromRandom("ed25519");
-      await createAccount(accountId, keyPair.getPublicKey());
+      const keyPair = await KeyPair.fromRandom();
+      await createAccount(accountId, keyPair.getPublicKey().toString());
```
Comment (vgrichina) [inline src/components/App.tsx]: why is this change needed?

Comment (willemneal) [reply src/components/App.tsx]: It wouldn't build for me otherwise.  The API in KeyPair was different.

**Comment by vgrichina** [comment]
Looks like this isn't needed anymore, correct? @willemneal

**Comment by willemneal** [comment]
Yeah this is very outdated.  I'm very glad to not have to worry about supporting this after moving to gitpod.
